### PR TITLE
Add face for product tshirts and face masks

### PR DESCRIPTION
**Pull request by antonkalachev**
Add face for product tshirts and face masks

### CODE REVIEW

File: src/InkRouter/Models/Face.php [hunk 1]
```diff
@@ -0,0 +1,473 @@
...
+ * @author Anton Kalachev
+ */
+class InkRouter_Models_Face
+{
+    /**
+     * Face name
+     *
+     * @var string
+     */
+//    private $pageNumber;
```
Comment (fightmaster) [inline src/InkRouter/Models/Face.php:22]: remove it

File: src/InkRouter/Models/Face.php [hunk 2]
```diff
@@ -0,0 +1,473 @@
...
+
+        return $this;
+    }
+
+//    /**
+//     * @return int
+//     */
+//    public function getPageNumber()
+//    {
+//        return $this->pageNumber;
```
Comment (fightmaster) [inline src/InkRouter/Models/Face.php:122]: remove

File: src/InkRouter/Models/OrderItem.php [hunk 3]
```diff
@@ -407,10 +442,21 @@ public function pack($root = false)
...
-        foreach ($this->sides as $side) {
-            $writer->writeRaw($side->pack());
+
+        if (!empty($this->faces)) {
+            $writer->startElement('faces');
+
+            foreach ($this->faces as $face) {
+                $writer->writeRaw($face->pack());
+            }
+        } else {
```
Comment (fightmaster) [inline src/InkRouter/Models/OrderItem.php:452]: if (!empty($this->sides)) {

File: src/InkRouter/Models/Face.php (hunk 1 above)
Comment (antonkalachev) [reply src/InkRouter/Models/Face.php]: removed

File: src/InkRouter/Models/Face.php (hunk 2 above)
Comment (antonkalachev) [reply src/InkRouter/Models/Face.php]: removed

File: src/InkRouter/Models/OrderItem.php (hunk 3 above)
Comment (antonkalachev) [reply src/InkRouter/Models/OrderItem.php]: fixed

**Pull request merged**